### DISCUSSION TITLE
[Feature Request]  Persist Battery Health custom property values (new battery capacity / max range)

### DISCUSSION THREAD

**Discussion by HenryMacGuy** [teslamate-org/teslamate#3728] [Ideas]
The custom properties for **new capacity and max range** on the **Battery Health** dashboard is helpful for a user who has owned a vehicle for a few years before using Teslamate.  However, putting these values in each time is inconvenient as they do not persist as a variable only in the dashboard.

A suggestion would be to allow the user to either save the values for persistent use or provide the values on the car setup, as these values will not generally change for a car once set.

<img width="981" alt="image" src="https://github.com/teslamate-org/teslamate/assets/123670537/8f706fc1-6ea1-4ab4-ac59-0e4da35ec432">

**Comment by StarkITA** [comment]
It would indeed be helpful, +1
